app/components/person/selector.js

```diff
@@ -16,16 +16,16 @@ export default class PersonSelectorComponent extends Component {
   @tracked creatingPerson = false;
   @tracked createPersonFormDefinition;
 
-  constructor() {
```

Comment:
Why did you remove this? This is still needed in other places where the selector is used, otherwise you never see the selected person when opening this component.
If this part is problematic for you, you should create another component, or add an option that circumvents the constructor if needed.
![Screenshot 2024-05-13 at 08 24 58](https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/43848896/05e5d7d4-40c3-4a46-8134-7a76fb92aa2f)